src/components/OnboardingGuide/Guide/Guide.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+
+import {Button, Icon, Progress} from '@gravity-ui/uikit';
+import {SVGIconData} from '@gravity-ui/uikit/build/esm/components/Icon/types';
+
+import {block} from '../../utils/cn';
+import i18n from '../i18n';
+
+import './Guide.scss';
+
+const cnGuide = block('guide');
+
+export type GuideProps = {
+    title: string | React.ReactNode;
+    progressBarCurrentValue: number;
+    icon?: SVGIconData;
+
+    defaultExpand: boolean;
+
+    isLoading?: boolean;
+
+    children?: React.ReactNode;
+
+    rollUpbuttonText?: string;
+    completeButtontext?: string;
+
+    onHeaderClick?: (newExpand: boolean) => void;
```

Comment:
If it's a click handler then it should have event in arguments. If you need expanded state, change the name to `onExpand: (expanded: boolean) => void`